src/plugins/cordova-plugin-network-information/sim-host.js

```diff
@@ -30,6 +37,18 @@ function initialize() {
     connectionTypeList.addEventListener('change', function () {
         network.updateNetworkType(connectionTypeList.value);
     });
+
+    registerTelemetryEvents();
+}
+
+function registerTelemetryEvents() {
+    var connectionList = document.getElementById('connection-list');
+
+    connectionList.onchange = function () {
+        var option = connectionList.options[connectionList.selectedIndex];
+
+        telemetry.sendUITelemetry(Object.assign({}, baseProps, { control: 'connection-list', value: option.value }));
+    };
```

Comment:
Thanks for your answer @guillaumejenkins! I wonder if it makes sense to keep track of the non-core plugins that doesn't support simulation in order to invest in adding it. Not for all of them but the most popular..